wolfcrypt/src/evp.c

```diff
@@ -3291,15 +3327,57 @@ int wolfSSL_EVP_PKEY_sign(WOLFSSL_EVP_PKEY_CTX *ctx, unsigned char *sig,
             *siglen = (size_t)len;
             return WOLFSSL_SUCCESS;
         }
-        /* wolfSSL_RSA_sign_generic_padding performs a check that the output
-         * sig buffer is large enough */
-        if (wolfSSL_RSA_sign_generic_padding(wolfSSL_EVP_MD_type(ctx->md), tbs,
-                (unsigned int)tbslen, sig, &usiglen, ctx->pkey->rsa, 1,
-                ctx->padding) != WOLFSSL_SUCCESS) {
-            return WOLFSSL_FAILURE;
+
+#if defined(OPENSSL_ALL) && (!defined(HAVE_FIPS) || FIPS_VERSION_GT(2,0))
```

Comment:
Would prefer way to enable this feature without having to force `OPENSSL_ALL` or gate on `WC_RSA_PSS`. Someone building without configure may not have `WC_RSA_PSS` enabled with compatibility layer.